package.json

```diff
@@ -69,7 +69,7 @@
     "jest-cli": "^23.6.0",
     "jest-prop-type-error": "^1.1.0",
     "node-sass": "^4.5.0",
-    "prettier": "^1.18.0",
+    "prettier": "^1.19.1",
```

Comment:
Should we use an exact version here so it doesn't break again when a new version gets released?

Reply:
IMO we benefit from their upgrades and if we will forget to upgrade from time to time,
the changes might be too big and hard to review.